int32 HumanID
float64[] RobotVelocity
float64[] WaitingTime
float64[] StartFilling
float64[] FinishFilling
float64[] StartServing
float64[] FinishServing
float64[] TimeFilling
float64[] TimeServing
int32 ConfirmServing
int32 ConfirmFilling
int32 Task
